src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -73,4 +73,8 @@
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="@MicrosoftWindowsSDKBuildToolsPackageVersion@" IsImplicitlyDefined="true" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true') and ('$(TargetPlatformIdentifier)' == 'tizen') ">
+    <PackageReference Include="Tizen.UIExtensions.ElmSharp" Version="@SamsungTizenUIExtensionsElmSharpPackageVersion@" IsImplicitlyDefined="true" PrivateAssets="all" />
```

Comment:
Not sure about PrivateAssets. We added that specifically since Microsoft.Windows.SDK.BuildTools is not meant to be referenced as it is build tools to set props. This is a real package.